### PR TITLE
DS-3167 empty list of metadataformat in JSPUI and XMLuUI

### PR DESCRIPTION
**Pull request by ghost**
https://jira.duraspace.org/projects/DS/issues/DS-3167

After the changes in the oai.cfg , these classes need some adjustments to not show up empty list of metadata format.

Best
Luiz

### CODE REVIEW

**Comment by LuizClaudioSantos** [comment]
I did a mess with the commits again..sorry, the commit 733960a is in another pull request (https://github.com/DSpace/DSpace/pull/1381), could you help me to eliminate this commit from this pull request?

best
Luiz

**Comment by DylanMeeus** [comment]
I believe this is also fixed in #1326

File: dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/administrative/collection/EditCollectionHarvestingForm.java [hunk 1]
```diff
@@ -139,7 +139,7 @@ public void addBody(Body body) throws WingException, SQLException, AuthorizeExce
 	    // The big complex way of getting to our metadata
 	    settings.addLabel(T_label_metadata_format);
     
-	    String key = "harvester.oai.metadataformats." + metadataFormatValue;
+	    String key = "harvester.metadataformats." + metadataFormatValue;
```
Comment (mwoodiupui) [inline dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/administrative/collection/EditCollectionHarvestingForm.java]: Should this be "oai.harvester.metadataformats"?

**Comment by tdonohue** [comment]
@LuizClaudioSantos : unfortunately we cannot fix the PR for you. It looks like you must have created your DS-3167 code branch based on your DS-3198 branch (for #1381).  So, if you recreated your DS-3167 code branch off of master, there shouldn't be a shared commit.

**Comment by luizfr** [comment]
Hi @tdonohue,  

Thanks for the explanation, so can you merge this pull request, because it have the two fixes:
 1. list of metadata format in jspui and xmlui.
 2. NullPointException in OAIHarvester.java

 The other pull request [(#1381)](https://github.com/DSpace/DSpace/pull/1381) could you just reject? 
Best 
Luiz

**Comment by tdonohue** [comment]
Note: Just realized this PR looks to be duplicated in the (much larger) fixes in #1326

**Comment by pnbecker** [comment]
I just merged #1326 and created #1428 which contains all from this PR that isn't resolved by #1326. I close this PR in favor for #1428.

**Pull request closed without merging**